### PR TITLE
docs(input): update the story to make the input interactive

### PR DESCRIPTION
**Pull request by veekays**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/input/__stories__/variants/types/BasicInput.story.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export const basic = () => {
   return (
     <div className="Row">
       <div className="mr-9 mb-8 w-25">
-        <Input name="input" value="Value" onChange={action('on-change')} onClear={action('on-clear')} />
+        <Input name="input" value="" onChange={action('on-change')} onClear={action('on-clear')} />
```
Comment (xlreon) [inline core/components/atoms/input/__stories__/variants/types/BasicInput.story.tsx]: Remove value

File: core/components/atoms/input/__stories__/variants/controlledInput.story.tsx [hunk 2]
```diff
@@ -44,13 +44,65 @@ export const controlledInput = () => {
...
+  const [value1, setValue1] = React.useState('Value');
+  const onChangeHandler = (event) => {
+    const updatedValue = event.target.value;
+    window.setTimeout(() => {
+      setValue1(updatedValue);
+    }, 1000);
+  };
+  const onClearHandler = () => {
+    const updatedValue = '';
+    window.setTimeout(() => {
```
Comment (xlreon) [inline core/components/atoms/input/__stories__/variants/controlledInput.story.tsx]: remove setTimeout

File: core/components/atoms/input/__stories__/variants/types/IconLeft.story.tsx [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const iconLeft = () => {
       <div className="mr-9 mb-8 w-25">
         <Input
           name="input"
-          value="Value"
+          value=""
```
Comment (xlreon) [inline core/components/atoms/input/__stories__/variants/types/IconLeft.story.tsx]: remove value

File: core/components/atoms/input/__stories__/variants/controlledInput.story.tsx [hunk 4]
```diff
@@ -44,13 +44,65 @@ export const controlledInput = () => {
...
+      <Input
+        name="input"
+        value={value}
+        type={visibility ? 'text' : 'password'}
+        placeholder="Placeholder"
+        actionIcon={
+          <Icon
+            onClick={() => setVisibility((x) => !x)}
+            name={visibility ? 'visibility_on' : 'visibility_off'}
+            aria-label={visibility ? 'Password Shown' : 'Password Hidden'}
```
Comment (xlreon) [inline core/components/atoms/input/__stories__/variants/controlledInput.story.tsx]: change aria-labels to Show password & Hide password

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**